picard/tagger.py

```diff
@@ -256,6 +256,33 @@ def __init__(self, picard_args, localedir, autoupdate, pipe_handler=None):
         if picard_args.debug or "PICARD_DEBUG" in os.environ:
             self.set_log_level(logging.DEBUG)
 
+        if sys.version_info[:3] > (3, 8):
```

Comment:
As @zas and I discussed yesterday we will drop Python 3.7 support after Picard 2.10.